app/src/main/java/com/example/myapplication/ThirdFragment.java

```diff
@@ -36,13 +43,47 @@ public class ThirdFragment extends Fragment {
 
 
     private FragmentThirdBinding binding;
+    ViewGroup v = null;
+
 
 
     @Override
     public View onCreateView(
             LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState
     ){
         binding = FragmentThirdBinding.inflate(inflater, container, false);
+        v = container;
+        if (binding.switch1.isChecked()){
```

Comment:
for `ColorStateList.valueOf(Color.parseColor("#6750A3"))` you're using it constantly, so when you have something like that it's best to save it into a variable or field in the class
Same with the other colors